ner/model.py

```diff
@@ -15,3 +34,6 @@ def load_data(sql):
 
     def placeholder_method(self, text):
         return text
+
+if __name__ == '__main__':
+    print(extract_entities('My name is Narendran Santhanam'))
```

Comment:
What is this for? :) can we move it elsewhere?

Reply:
!!! I have no idea why I didn't see that before pushing it *cringes*